### DISCUSSION TITLE
Minimal implementation like @xstate/fsm for v5?

### DISCUSSION THREAD

**Discussion by jaskp** [statelyai/xstate#4869] [Q&A]
Is there any maintained alternative to @xstate/fsm? I'd like to use a small portion of XState's functionality.

**Comment by davidkpiano** [comment]
You can still see the old implementation here: https://github.com/statelyai/xstate/blob/v4/packages/xstate-fsm/src/index.ts

**Comment by jacksteamdev** [comment]
Is @xstate/store the v5 replacement for @xstate/fsm?

**Reply by davidkpiano** [reply]
It is not - `@xstate/store` is for simple storage, not for finite states.
